src/main/java/edu/ucsb/nceas/mdqengine/dispatch/Dispatcher.java

```diff
@@ -197,4 +272,36 @@ public Map<String, Object> getBindings() {
 	public void setBindings(Map<String, Object> bindings) {
 		this.bindings = bindings;
 	}
```

Comment:
Method `setBindings(java.util.Map<java.lang.String,java.lang.Object>)` is never used